tests/integration/test_api_gateway.py

```diff
@@ -127,3 +148,40 @@ def forward_request(self, **kwargs):
 
     # clean up
     proxy.stop()
+
+
+# def test_api_gateway_lambda_proxy_integration():
```

Comment:
Sure, no worries. We can merge the PR as-is for now. I think I can actually take it over from here and complete the work on this test method. 👍 

Maybe one more thing. Could you please rebase/squash the 7 commits into a single commit? This would help keep the commit history clean. There is also a feature to squash-merge PRs on Github, but the problem is that this will merge all commit comments into one, which is not really desirable Thanks.